LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright © 2013 Rod Vagg and the contributors to abstract-level and abstract-leveldown.
+Copyright © 2013 Rod Vagg and the contributors to abstract-level.
```

Comment:
I removed the `and abstract-leveldown` because I'd also have to list `encoding-down`, `levelup`, `deferred-leveldown` and `subleveldown`. Not sure how meaningful that is.

@mafintosh I copied code from `subleveldown` to here. You're the [copyright holder on `subleveldown`](https://github.com/Level/subleveldown/blob/f132be81a01fd1260f3bc4e8801185707154a4a7/LICENSE#L3). Do you want your name in the `LICENSE` file of `abstract-level` (or in code comments)?